elements/rh-footer/rh-footer-lightdom.css

```diff
@@ -1,48 +1,51 @@
+/* Via pfe-base.css */
+:is(rh-footer, rh-global-footer) a {
+  color: var(--rh-color-link-inline-on-dark, var(--rh-color-blue-200, #73bcf7));
+  text-decoration: none;
+}
+
+:is(rh-footer, rh-global-footer) a:hover {
+  color: var(--rh-color-link-inline-hover-on-dark, var(--rh-color-blue-100, #bee1f4));
+  text-decoration: underline;
+}
+
+:is(rh-footer, rh-global-footer) a:is(:focus, :focus-within) {
+  color: var(--rh-color-link-inline-focus-on-dark, var(--rh-color-blue-100, #bee1f4));
+  text-decoration: underline;
+}
+
+:is(rh-footer, rh-global-footer) a:visited {
+  color: var(--rh-color-link-inline-visited-on-dark, var(--rh-color-blue-100, #bee1f4));
+  text-decoration: none;
+}
+
 /* ensure links fully wrap img tags */
-:is(rh-footer, rh-global-footer) a[slot^=logo] {
+:is(rh-footer, rh-global-footer) a[slot^="logo"] {
   display: block;
-  height: 36px;
+  height: var(--rh-size-icon-04, 40px);
 }
 
-:is(rh-footer) a[slot^=logo] > img {
+:is(rh-footer) a[slot^="logo"] > img {
   width: auto;
   height: 100%;
 }
 
-rh-footer [slot^=links] a {
-  gap: var(--rh-footer-links-gap, 0.625em);
+rh-footer [slot^="links"] a {
+  gap: var(--rh-footer-links-gap, var(--rh-space-2xs, 8px));
 }
 
-:is(rh-footer, rh-global-footer) [slot^=links] li {
+:is(rh-footer, rh-global-footer) [slot^="links"] li {
   margin: 0;
   padding: 0;
   display: contents;
 }
 
-:is(rh-footer, rh-global-footer) [slot^=links] a {
+:is(rh-footer, rh-global-footer) [slot^="links"] a {
   display: block;
-  color: var(--link-color, #fff) !important;
-  font-size: var(--link-font-size, 0.875em);
+  color: var(--rh-color-text-primary-on-dark, #ffffff) !important;
+  font-size: var(--rh-footer-link-font-size, var(--rh-font-size-body-text-sm, 0.875rem));
```

Comment:
Following up on https://github.com/orgs/RedHat-UX/discussions/385#discussioncomment-3523950 here,

![screenshot](https://user-images.githubusercontent.com/45767595/187850448-2e446881-b558-40a3-985b-1a692c9d79f1.png)

@coreyvickery can you tell me which specific token name is preferred here?

The token `--rh-font-size-body-text-sm` has the value `0.875rem`, which [corresponds](https://codebeautify.org/rem-to-px-converter) to `14px` at a 16px base.

Because the font size is defined in rems, it will adjust depending on your browser font size settings or based on the page font size.

If this is in fact the right size, but is rendering contrary to expectations due to user settings or page root font size, we can adopt ems instead, but I think that's outside the scope of this PR, and we'd have to take it up in the discussion thread above.

Alternately, we can provide a non-token font size here, but your earlier comments in this thread advised against that.

Reply:
@coreyvickery I pushed a commit just now which sets the site-wide base font size to 16px across ux-dot, including the demos.

Please check again, as I suspect this will solve your issue.

![image](https://user-images.githubusercontent.com/1466420/187861523-fcf86de4-c1c0-430b-bd2f-4b7bba06b1a6.png)

Reply:
@bennypowers The demo looks good, thank you for adjusting.